acm-pca-cert-generator.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [ "${INTERNET_PROXY}" ]; then
+  export http_proxy="http://${INTERNET_PROXY}:3128"
+  export https_proxy="https://${INTERNET_PROXY}:3128"
+  export no_proxy=169.254.169.254
+  echo "Using proxy ${INTERNET_PROXY}"
+fi
+
+echo "Generating cert for host ${HOSTNAME}"
+acm-pca-cert-generator --subject-cn "${HOSTNAME}" 
+echo "Cert generation result for ${HOSTNAME} is: $?"
```

This echo will make the script always exit with a success status, masking any failure of the ACM generator script. Is that intended, or do you need to add an `exit $acm_exit_status` type thing at the end?

---

This code wouldn't prevent Kafka consumer from starting if certificate generation failed which _I think_ is preferred in the interim so we could switch to the new docker image to test certificate generation but not affect existing non-SSL broker/consumer communication.
When work to configure broker and consumer to use SSL will be complete it would be worth it to change this script to fail on certificate generation so that consumer wouldn't try to start with no certificate.

---

That makes sense. Thanks.